add quick-add-key command

kilimnik opened the pull request:
I implemented the already discussed feature request in #164.

I am unsure if the tests are enough. I am happy to write more if necessary.

kilimnik committed:
add quick-add-key command

kilimnik committed:
add generate subkey documentation

vsajip reviewed (changes requested):
Thanks very much for this. As well as the comments I've made on the PR, what about testing use of the subkey? Does it show up when keys are listed, etc. ? What about deleting a subkey? It would be worth adding tests for these.

kilimnik committed:
add subkey requested changes

change naming of method
add more tests
add versionadded directive to docs
add exclamation_mode argument to deletion, to allow for delting specific subkeys

vsajip reviewed (changes requested):
Thanks for the changes. Just one more minor change, please, relating to Python 2.7 compatibility.

kilimnik committed:
fix python27 compatibility
